.github/workflows/ci.yaml

```diff
@@ -21,7 +21,20 @@ jobs:
         buildtype: [release, debug]
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout HSE
+        uses: actions/checkout@v2
+
+      - name: Checkout hse-python
+        uses: actions/checkout@v2
+        with:
+          repository: hse-project/hse-python
+          path: subprojects/hse-python
+
+      - name: Post-checkout
+        run: |
+          for b in hse-python; do
+            git -C "subprojects/$b" checkout ${{ github.head_ref }} || true
+          done
```

Comment:
Why in Post-checkout instead of in "Checkout hse-python"?

Reply:
Because the branch name might not exist, in which case, I don't want the step to fail

Reply:
Aha.  Hence the `|| true`